HomeWork/HomeWork/UserService.cs

```diff
@@ -0,0 +1,578 @@
+        private readonly By _audioPageButton = By.XPath("//a[@href='/k10.htm' and text()='Аудио']");
+        private readonly By _audioPageDropButton = By.XPath("//li[@class='mainmenu-item']//a[text()='Аудио']//following-sibling::div//a[@href='/k239.htm']");
+        private readonly By _audioBrandFilter = By.XPath("//label[@for='br124']");
+        private readonly By _saveListButton = By.XPath("//span[text()='Сохранить список']");
+        private readonly By _acceptSaveListButton = By.XPath("//button[@type='submit']");
+        private readonly By _showSaveList = By.XPath("//a[@class='user-menu__section wu_bookmarks ']");
+        private readonly By _nameBrandSaveList = By.XPath("//span[@class='u' and text()]");
+        private readonly By _consoleDropButton = By.XPath("//li[@class='mainmenu-item']//a[text()='Компьютеры']//following-sibling::div//a[@href='/k33.htm']");
+        private readonly By _filterOnConsolePage = By.XPath("//label[@class='brand-best' and @for='br156']");
+        private readonly By _nameConsoleItem = By.XPath("//span[@class='u' and text()='Sony PlayStation 5']");
+        private readonly By _nameAudioItem = By.XPath("//span[@class='u' and text()='Logitech G Pro X']");
+        private readonly By _searchingItems = By.XPath("//td[@class='where-buy-description']//h3[text()]");
+
+
+        private const string _registrationName = "User1";
+        private const string _registrationEmail = "danya.sydortsov@tech-stack.io";
+        private const string _registrationPassword = "Password123";
+        private const string _searchingItem = "iPhone 13 Pro 256";
+
+        //public By Computer => By.XPath("//a[@class='cancel-click mainmenu-link']");
+        public By ToCompareTablet => By.XPath("//label[@id='label_2090044']");
+
+        RandomUser rndUs = new RandomUser();
+
+
+        public void Create()
+        {
+            var rndLg = rndUs.CreateRandomLogin();
+
+            var signIn = driver.FindElement(_signInButton);
+            signIn.Click();
+
+            Thread.Sleep(2000);
+
+            var registrationButton = driver.FindElement(_registrationButton);
+            registrationButton.Click();
+
+            Thread.Sleep(1000);
+
+            var inputName = driver.FindElement(_nameInputButton);
+            inputName.SendKeys(rndLg);
+
+            var emailInput = driver.FindElement(_emailInputButton);
+            emailInput.SendKeys(rndUs.CreateRandomEmail());
+
+            var passwordInput = driver.FindElement(_passwordInputButton);
+            passwordInput.SendKeys(rndUs.CreateRandomPassword());
+
+            var acceptRegistration = driver.FindElement(_acceptRegistrationButton);
+            acceptRegistration.Click();
+
+            Thread.Sleep(2000);
+
+            var acceptButton = driver.FindElement(_acceptButton);
+            acceptButton.Click();
+
+            var actualLogin = driver.FindElement(_acceptLogin).Text;
+
+            Assert.AreEqual(actualLogin, rndLg, "Wrong!");
```

Comment:
we dont add symbols like '?' or '!' to error messages, please make error message more informative